src/common_modules/vsts-cli-package-common/vsts/cli/package/common/upack.py

```diff
@@ -0,0 +1,64 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import logging
+
+import colorama
+from vsts.cli.common.exception_handling import handle_command_exception
+from vsts.cli.common.services import get_core_client, resolve_instance
+
+from .artifacttool import ArtifactToolInvoker
+from .artifacttool_updater import ArtifactToolUpdater
+from .external_tool import ProgressReportingExternalToolInvoker
+
+logger = logging.getLogger('vsts.packaging')
+
+def publish_package(feed, name, version, path, description=None, team_instance=None, detect=None):
+    """Publish a package to a feed.
+    :param feed: Name or ID of the feed
```

Comment:
nit pick for consistency, we put a period at the end of each comment. Same standard as azure cli.